web-ui/src/components/member_selector/member_selector_dialog/MemberSelectorDialog.jsx

```diff
@@ -414,11 +443,11 @@ const MemberSelectorDialog = ({ open, selectedMembers, onClose, onSubmit }) => {
                   {...params}
                   variant="outlined"
                   label="Filter Members"
-                  placeholder={`Search for ${filterType.toLowerCase()}`}
+                  placeholder={`Search for ${filterType?.toLowerCase()}`}
                 />
               )}
               disablePortal
-              disabled={!filterOptions}
+              disabled={!filterOptions || initialFilters.length > 0}
```

Comment:
This is the `Autocomplete` component, which will throw if it receives `0` as an argument.